addons/mail/static/src/models/composer/composer.js

```diff
@@ -287,9 +298,48 @@ function factory(dependencies) {
             body = this._generateMentionsLinks(body);
             body = parseAndTransform(body, addLink);
             body = this._generateEmojisOnHtml(body);
+            return body;
+        }
+
+        /**
+         * Recursive function that process the queue.
+         */
+        async processMessageQueue() {
+            if (!this.isProcessingQueue && !this.isQueueError) {
+                try {
+                    this.update({
+                        isProcessingQueue: true,
+                    });
+                    const message = this.thread.queue[0];
+                    await this._postNextMessageInQueued(message);
+                    this.thread.update({ queue: [['unlink', message]] });
```

Comment:
All of this should be part of `_postNextMessageInQueued` probably, no need to play around with parameter.